cmd/osbuild-image-tests/main_test.go

```diff
@@ -92,39 +94,27 @@ func splitExtension(path string) (string, string) {
 
 // testImageInfo runs image-info on image specified by imageImage and
 // compares the result with expected image info
-func testImageInfo(imagePath string, rawImageInfoExpected []byte) error {
+func testImageInfo(t *testing.T, imagePath string, rawImageInfoExpected []byte) {
 	var imageInfoExpected interface{}
 	err := json.Unmarshal(rawImageInfoExpected, &imageInfoExpected)
-	if err != nil {
-		return fmt.Errorf("cannot decode expected image info: %v", err)
-	}
+	require.Nil(t, err)
 
 	cmd := exec.Command(imageInfoPath, imagePath)
 	cmd.Stderr = os.Stderr
 	reader, writer := io.Pipe()
 	cmd.Stdout = writer
 
 	err = cmd.Start()
-	if err != nil {
-		return fmt.Errorf("image-info cannot start: %v", err)
-	}
+	require.Nil(t, err)
```

Comment:
Same here, looks like you've removed error messages everywhere. IMO it is better for them to stay.